api/rosetta/errors.go

```diff
@@ -23,19 +23,21 @@ import (
 const (
 	invalidJSON = "request does not contain valid JSON-encoded body"
 
-	blockchainEmpty = "blockchain identifier has empty blockchain field"
-	networkEmpty    = "blockchain identifier has empty network field"
-	blockEmpty      = "block identifier has empty index and hash fields"
-	blockLength     = "block identifier has invalid hash field length"
-	addressEmpty    = "account identifier has empty address field"
-	addressLength   = "account identifier has invalid address field length"
-	currenciesEmpty = "currency identifier list is empty"
-	symbolEmpty     = "currency identifier has empty symbol field"
-	txHashEmpty     = "transaction identifier has empty hash field"
-	txLength        = "transaction identifier has invalid hash filed length"
-	txInvalidOps    = "transaction operations are invalid"
-	txBodyEmpty     = "transaction text is empty"
-	txBodyInvalid   = "transaction text is invalid"
+	blockchainEmpty   = "blockchain identifier has empty blockchain field"
+	networkEmpty      = "blockchain identifier has empty network field"
+	blockEmpty        = "block identifier has empty index and hash fields"
+	blockLength       = "block identifier has invalid hash field length"
+	addressEmpty      = "account identifier has empty address field"
+	addressLength     = "account identifier has invalid address field length"
+	currenciesEmpty   = "currency identifier list is empty"
+	symbolEmpty       = "currency identifier has empty symbol field"
+	txHashEmpty       = "transaction identifier has empty hash field"
+	txLength          = "transaction identifier has invalid hash filed length"
+	txInvalidOps      = "transaction operations are invalid"
+	txBodyEmpty       = "transaction text is empty"
+	txBodyInvalid     = "transaction text is invalid"
+	txSignaturesEmpty = "transaction signature list is empty"
+	txSignaturesFound = "unsigned transaction has non-empty signature list"
```

Comment:
This deviates from the pattern, but is less clunky than the alternatives that I thought of.